docs/MockFunctionAPI.md

```diff
@@ -196,7 +196,7 @@ Will result in this error:
 
 ### `mockFn.mockReturnThis()`
 
-Just a simple sugar function for:
+Just a syntactic sugar function for:
```

Comment:
```suggestion
Just syntactic sugar function for:
```

a few different places as well

Reply:
Thanks for the suggestion! I've updated the phrasing to `Syntactic sugar function for:` to match usage throughout the rest of the file.